src/components/PuzzleModal/index.scss

```diff
@@ -0,0 +1,66 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.puzzlemodal__container {
+  position: absolute;
+  margin-left: auto;
+  margin-right: auto;
+  left: 0;
+  right: 0;
+  width: 200px;
+  display: flex;
```

Comment:
Let's increase the size of the modal just a little bit (maybe 5-10% bump in height and width.